Master Project/Assets/Scenes/Main/IngredientDirectiveScript.cs

```diff
@@ -27,17 +28,16 @@ public class IngredientDirectiveScript : MonoBehaviour {
 		//Debug.Log ("Collision");
 		if (col.gameObject.tag == "ItemOne") {

```

Comment:
None of this is in a method... This won't compile.

Reply:
Woah that’s pretty weird it’s in a method for me idk why it’s doing that